build.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+[[ -f /etc/prfile.d/modules.sh ]] && \
+  source /etc/profile.d/modules.sh
+
+builddir=build
+[ -d $builddir ] && rm -rf $builddir; mkdir $builddir
+
+if type module
+then
+  proj_dir=/qfs/projects/exasgd
+  mod_dir=/share/apps
+  cluster=$(uname -n | cut -d'.' -f1 | sed -E 's/[[:digit:]]//g')
+  module purge
+  if [ "$cluster" == "newell" ]
+  then
+    gcc=7.4.0
```

Comment:
Perhaps user has loaded modules on a machine that is not available as an option in this script, so she does not want to purge modules. Try something like:
```suggestion
  cluster=$(uname -n | cut -d'.' -f1 | sed -E 's/[[:digit:]]//g')
  if [ "$cluster" == "newell" ]
  then
    proj_dir=/qfs/projects/exasgd
    mod_dir=/share/apps
    module purge
    gcc=7.4.0
```

Reply:
The modules will only be purged within the environment of the script, not in the user's environment. If the user's build needs a new module loaded, that should be part of the PR as well I think. Not purging modules could shadow a new dependency that would not show up in the build script but would fail on other dev's devices.